packages/slice-machine/server/src/api/tracking.ts

```diff
@@ -16,11 +17,18 @@ export function sendEvents(
   userId?: string,
   intercomHash?: string
 ): void {
+  const pathToPackage = path.resolve(__dirname, "..", "..", "..");
+  const pkg = retrieveJsonPackage(pathToPackage);
+  const version = pkg.content?.version ?? "";
```

Comment:
Do we need to do this on every event? Why not just once when the module loads?

Reply:
Can do :)